### ISSUE TITLE
Error of ConfigurationError on Plone 6

### ISSUE THREAD

**Issue by terapyon** [collective/collective.cloudfront#2]
I got an error as below on Plone 6.

```
zope.configuration.exceptions.ConfigurationError: ('Unknown directive', 'http://namespaces.zope.org/zope', 'includeDependencies')
...
../eggs/cp38/collective.cloudfront-1.3-py3.8-macosx-10.15-x86_64.egg/collective/cloudfront/configure.zcml", line 10.2
```

Do yon know how to fix it?

**Comment by terapyon** [comment]
I found below the document.
https://community.plone.org/t/includedependencies-package-what-does-it-do-plone-6/14780

I think no need to set a dependencies.

If yes, remove the `<includeDependencies package="." />`
https://github.com/collective/collective.cloudfront/blob/master/src/collective/cloudfront/configure.zcml#L10

If no, modify the ZCML to below.

```
<include package="plone.behavior" />
```

**Comment by rber474** [comment]
Removed includeDependencies as it's obsolete in Plone6

**Comment by rber474** [comment]
Doing things properly, awaiting for the pull request being reviewed.

**Comment by erral** [comment]
Fixed by #4
